source/reporting/index.html.md.erb

```diff
@@ -166,6 +165,24 @@ If you search for a specific payment, all criteria you use must match. Otherwise
 
 If your request has no query parameters, the API will return all payments.
 
+#### Search by card brand
+
+If you search for payments by `card_brand`, the value you need to use is:
+
+- case insensitive and exact match only
+- different to the value in the API response for some card brands
+
+|`card_brand` to search for|`card_brand` in API response|
+|-|-|
+|`visa`|`Visa`|
+|`master-card`|`Mastercard`|
+|`maestro`|`Maestro`|
+|`american-express`|`AmericanExpress`|
```

Comment:
The API response is actually “American Express”

Reply:
Ah thank you @alexbishop1 - I did a blanket search-and-replace for spaces in the table, which is where all the spaces in the values went 😊